app/Console/Commands/InitCommand.php

```diff
@@ -172,12 +169,22 @@ private function maybeSetMediaPath(): void
             return;
         }
 
-        $this->info('The absolute path to your media directory. If this is skipped (left blank) now, you can set it later via the web interface.');
+        if ($this->inNoInteractionMode()) {
+            $path = config('koel.media_path');
+
+            if (is_dir($path) && is_readable($path)) {
+                Setting::set('media_path', $path);
+                return;
+            }
+
+            $this->error('The path '.$path.' does not exist or not readable. Skipping.');
```

Comment:
We can return early here :)

Reply:
just added a `return` statement after the error message